app/models/contact.rb

```diff
@@ -2,10 +2,16 @@ class Contact < ApplicationRecord
   RELATIONSHIP_TYPES = %w[friend family colleague]
 
   belongs_to :user
+
   has_many :interactions, dependent: :destroy
 
-  validates :name, :user_id, :phone_number, :relationship, :address, presence: true
-  validates :email, uniqueness: true
-  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP, message: "Must be a valid email address 📪" }
-  validates :relationship, inclusion: { in: RELATIONSHIP_TYPES, message: "%{value.capitalize} is not a valid relationship 🕺" }
+  validates :user_id, presence: true
+  validates :name, :relationship,  presence: true
+  validates :name, :relationship, :email, :phone_number, :address, :social_media_handle, presence: true
+  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP, message: 'Must be a valid email address 📪' }
+  validates :relationship, inclusion: { in: %w[friend family colleague], message: "%{value} is not a valid relationship" }
+  
+  def self.relationship_options
+    [:friend, :family, :colleague]
+  end
```

Comment:
are we using self. for readability?

Reply:
Also dont we either need this or the Constant Array at the top? Or is there a reason for both?🎈